src/D2L.Security.OAuth2/Keys/Default/JsonWebKeySet.cs

```diff
@@ -39,7 +39,7 @@ internal sealed class JsonWebKeySet {
 			m_keys = ImmutableArray.Create( jsonWebKey );
 		}
 
-		public bool TryGetKey( Guid keyId, out JsonWebKey key ) {
+		public bool TryGetKey( string keyId, out JsonWebKey key ) {
 			foreach( JsonWebKey currentKey in m_keys ) {
 				if( currentKey.Id == keyId ) {

```

Comment:
I believe strict equality should be safe here.

Reply:
Yes, I do believe this does `string.Equals( a, b )` which defaults to `Ordinal`, but I'd be more comfortable if it was called out specifically.

Either `string.Equals( a, b, StringComparison.Ordinal )`, or perhaps a static "keyid comparer" utility function so we don't have to be sure to get it right everywhere?

Reply:
- [X] string KeyId comparison helper